spec/views/two_factor_authentication/otp_verification/show.html.erb_spec.rb

```diff
@@ -313,14 +293,20 @@
         otp_expiration = user.direct_otp_sent_at +
                          TwoFactorAuthenticatable::DIRECT_OTP_VALID_FOR_SECONDS
         allow(@presenter).to receive(:otp_expiration).and_return(otp_expiration)
-        allow(IdentityConfig.store).to receive(:allow_otp_countdown_expired_redirect).
-          and_return(true)
       end
 
-      it 'should render countdown component with redirect-url' do
+      it 'should render countdown component' do
         render
-        expect(rendered).to have_selector(
-          "lg-countdown-alert[redirect-url='#{login_two_factor_otp_expired_path}']",
+
+        expect(rendered).to have_content(
+          t(
+            'components.countdown_alert.time_remaining_html',
+            countdown_html: distance_of_time_in_words(
+              Time.zone.now,
+              Time.zone.now + TwoFactorAuthenticatable::DIRECT_OTP_VALID_FOR_SECONDS,
```

Comment:
if we wanted to use some activesupport magic while we're here...
```suggestion
              TwoFactorAuthenticatable::DIRECT_OTP_VALID_FOR_SECONDS.seconds.from_now,
```

Reply:
> if we wanted to use some activesupport magic while we're here...

I'd really hoped there was some way to just convert the seconds value to human-readable distance, but alas it doesn't appear so. The approach here followed pretty strictly to [the examples in `dotiw` documentation](https://github.com/radar/distance_of_time_in_words#distance_of_time_in_words).

But yeah, I think the ActiveSupport magic lends some readability.